src/components/info-card/info-card.styles.js

```diff
@@ -0,0 +1,47 @@
+import theme from '~/styles/app-theme/custom-mui.styles'
+
+export const cardStyles = {
+  card: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    boxSizing: 'border-box',
+    maxWidth: {
+      md: '427px',
+      xs: '343px',
+    },
+    padding: '24px 24px 32px',
+    backgroundColor: theme.palette.basic.white,
+    boxShadow: theme.shadows.primary,
+    borderRadius: '6px',
+    '&:not(:last-child)': {
```

Comment:
I use this pseudo classes for adding indent between cards. It's fine or i should look for alternate solution?

Reply:
Fixed.